src/output/outputMod.F90

```diff
@@ -1782,17 +1832,119 @@ subroutine storeSurfsolInBuffer(sps, buffer, nn, blockID,   &
        !================================================================
 
     case (cgnsCp)
-
+       ! mham:
```

Comment:
you can remove the comments between line 1835 - 1843, these are from when mads was figuring this stuff out